pkg/util/test/util.go

```diff
@@ -646,6 +661,44 @@ func validateComponents(pxImageList map[string]string, cluster *corev1.StorageCl
 		return err
 	}
 
+	if cluster.Spec.Placement != nil {
```

Comment:
Let's fix this as discussed on the call.

Reply:
Done